src/test/integration/configuration_example.json

```diff
@@ -0,0 +1,7 @@
+{
+    "hostname": "https://api.example.com",
```

Comment:
Could we make this example hostname more useful? Something like `https://api.example.com/api/v1/locations`